Modules/vc-module-orders/VirtoCommerce.OrdersModule.Data/Model/OperationEntity.cs

```diff
@@ -32,10 +32,15 @@ public abstract class OperationEntity : AuditableEntity
         [StringLength(2048)]
         public string CancelReason { get; set; }
 
+        [NotMapped]
```

Comment:
Probably it is not needed. The main concern here and for all changed models is the following:
1. Make all model decimal properties which were reset in v2 nullable (decimal?).
2. When updating entities from model, check whether model value is null. If null - do not assign the property from model, otherwise - assign.
3. `ResetPrices` method implementations should assign null to properties, not 0m.

Thus we eliminate all these:
```
var isNeedPatch = !(GetNonCalculatablePrices().All(x => x == 0m) && target.GetNonCalculatablePrices().Any(x => x != 0m));
```
```
public bool NeedPatchSum { get; set; } = true;
```
```
if (isNeedPatch)
{
…
}
```
```
public virtual IEnumerable<decimal> GetNonCalculatablePrices()
```
And remove `GetNonCalculatablePrices` from `ISupportPartialPriceUpdate`.

Please contact me on details.